.github/workflows/smoke-tests-heroku.yml

```diff
@@ -0,0 +1,45 @@
+name: Smoke Tests Nightly -Heroku
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Check designer container is running
+        run: |
+          curl -4 --retry 10 --retry-connrefused --retry-delay 10 https://digital-form-builder-designer.herokuapp.com/health-check
+      - name: Check runner container is running
+        run: |
+          curl -4 --retry 10 --retry-connrefused --retry-delay 10 https://digital-form-builder-runner.herokuapp.com/health-check
+      - name: Smoke Test
+        run: |
+          yarn smoke-tests/designer smoke-test-headless --baseUrl=https://digital-form-builder-designer.herokuapp.com/
```

Comment:
I don't think we need to check these are running. If the services aren't running the smoke tests will detect that it can't access them

Reply:
Thanks, I have pushed an update